### PR TITLE
NVMe driver: add v6.8 benchmarks

### PR DESCRIPTION
**Pull request by metaspace**

### CODE REVIEW

File: src/NVMe-driver.md [hunk 1]
```diff
@@ -23,6 +23,25 @@ The driver is not currently suitable for general use.
...
+
+### Results
+
+- 40 samples
+- Difference of means modeled with t-distribution
+- P95 confidence intervals
+
+![](rnvme/nvme-v6.8-absolute.svg)
+
+![](rnvme/nvme-v6.8-relative.svg)
```
Comment (alex) [inline src/NVMe-driver.md:43]: It may be helpful to label "lower means C is faster"

Comment (metaspace) [reply src/NVMe-driver.md:43]: I pushed an explanation. @ojeda please squash it all when merging.

File: src/rnvme/nvme-v6.8-absolute.svg
Comment (y86-dev) [inline src/rnvme/nvme-v6.8-absolute.svg:1]: In the plot legend there is a typo: "Cofniguration". Also I think there are no spaces after the commas, maybe adding them makes it easier to read.

Comment (metaspace) [reply src/rnvme/nvme-v6.8-absolute.svg:1]: Thanks, fixed

**Comment by ojeda** [comment]
Hmm... I was going to upload it, but it seems MathJax, when it renders the final form (rather than the preview it also does), does not respect the theme. So darker themes make the output (black) essentially unreadable. Can you reproduce it?

I think, for something as simple as this, it may be best to extract the HTML it generates (the preview form) and copy-paste a simplified form inline, and removing the dependency.

It also seems to take a very long time to generate the final form, which isn't great either.

What do you think?

**Comment by metaspace** [comment]
> Hmm... I was going to upload it, but it seems MathJax, when it renders the final form (rather than the preview it also does), does not respect the theme. So darker themes make the output (black) essentially unreadable. Can you reproduce it?

@ojeda no, it works with all 5 themes available in my local copy. I'm on `mdbook` 0.4.37.

**Comment by ojeda** [comment]
Strange. Can you please checkout the CI-generated branch and see if you can see it with that one? If yes, it is likely the version (we have v0.4.25 in the CI). If not, then I can take a look at my browser... :)

**Comment by metaspace** [comment]
I checked out the generated artifacts in the `gh-pages` branch. The equation from that place renders correctly on my computer in both Firefox and Chromium, in all the themes. I think it is using some kind of JS to type set it, because I get a special menu if I left click the fraction. Did you disable JS or something?

**Comment by ojeda** [comment]
Thanks Andreas -- I had JS enabled, but the issue were the fonts (which I block). Apparently MathJax loads a few of them and, if that fails, fallbacks (in the default configuration) after a few *seconds* into an image-based approach which does not respect the theme (and still using the same default renderer). Any of the other renderers would actually work fine.

Anyway, for something like this, I think the best compromise is just using MathML -- we can be modern and avoid the dependency :)

So I have just done that and rebased it. Please let me know what you think (it is generated now) and I can publish it if you are happy with it.

**Comment by metaspace** [comment]
Looks fine to me :+1: I did not know of this MathML thing!

**Comment by ojeda** [comment]
Thanks for double-checking! It is live now.

I think I remember reading about MathML (or similar) a long time ago, but it was not supported everywhere. But, now in 2024, apparently it is ready, which is great.

**Pull request merged**